templates/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Colorado Gold Rust</title>
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+  <link href="{{ get_url(path="style.css") }}" rel="stylesheet">
+</head>
+<body>
+  <header id="top">
+    <img class="flag" src="{{ resize_image(path="logo.png", width=300, op="fit_width") }}">
+    <h1>Colorado Gold Rust</h1>
+  </header>
+  <p class="headline">Join us in Denver, Colorado on September 20-21, 2018, for two days of rowdy Rust in the wild west.</p>
```

Comment:
This might not be grammatical. I think it is more aesthetically pleasing though.
This is tagline for the conference as it stands right now so I think it's fine to capitalize it for that reason, as well.
```suggestion
  <p class="headline">Join us in Denver, Colorado on September 20-21, 2019, for two days of Rowdy Rust in the Wild West.</p>
```